build/Targets/GenerateVersionInfoResourceFile.targets

```diff
@@ -42,6 +42,7 @@ BEGIN
       VALUE "LegalCopyright",   "© Microsoft Corporation. All rights reserved."
       VALUE "FileVersion",      "$(BuildVersion)"
       VALUE "ProductVersion",   "$(BuildVersion)"
+      VALUE "Comments",         "$(GitHeadSha)"
```

Comment:
You don't have to update this; this is for C++ which I'm outright deleting in #12151. (You might also want to be more generally aware of that PR.)